src/mc_robots/jvrc1.h

```diff
@@ -14,7 +14,35 @@ namespace mc_robots
 struct MC_ROBOTS_DLLAPI JVRC1RobotModule : public mc_rbdyn::RobotModule
 {
 public:
-  JVRC1RobotModule();
+  JVRC1RobotModule(bool fixed = false);
 };
 
 } // namespace mc_robots
+
+extern "C"
+{
+  ROBOT_MODULE_API void MC_RTC_ROBOT_MODULE(std::vector<std::string> & names)
+  {
+    names = {"JVRC1", "JVRC1Fixed"};
+  }
+  ROBOT_MODULE_API void destroy(mc_rbdyn::RobotModule * ptr)
+  {
+    delete ptr;
+  }
+  ROBOT_MODULE_API mc_rbdyn::RobotModule * create(const std::string & name)
+  {
+    if(name == "JVRC1")
+    {
+      return new mc_robots::JVRC1RobotModule(false);
+    }
+    else if(name == "JVRC1Fixed")
+    {
+      return new mc_robots::JVRC1RobotModule(true);
+    }
+    else
+    {
+      mc_rtc::log::error("JVRC1 module Cannot create an object of type {}", name);
+      return nullptr;
+    }
+  }
+}
```

Comment:
This breaks `MC_RTC_BUILD_STATIC`, you can have (to be passed through clang-format):

```cpp
#ifndef MC_RTC_BUILD_STATIC
extern "C"
{
  ROBOT_MODULE_API void MC_RTC_ROBOT_MODULE(std::vector<std::string> & names)
  {
    names = {"JVRC1", "JVRC1Fixed"};
  }
  ROBOT_MODULE_API void destroy(mc_rbdyn::RobotModule * ptr)
  {
    delete ptr;
  }
  ROBOT_MODULE_API mc_rbdyn::RobotModule * create(const std::string & name)
  {
    if(name == "JVRC1")
    {
      return new mc_robots::JVRC1RobotModule(false);
    }
    else if(name == "JVRC1Fixed")
    {
      return new mc_robots::JVRC1RobotModule(true);
    }
    else
    {
      mc_rtc::log::error("JVRC1 module Cannot create an object of type {}", name);
      return nullptr;
    }
  }
}
#else
namespace                                                                          
{                                                                                  
    static auto registered = []() {                                                    
      using fn_t = std::function<JVRC1RobotModule *()>;                                            
      mc_rbdyn::RobotLoader::register_object("JVRC1", fn_t([]() { return new JVRC1RobotModule(false); })); 
      mc_rbdyn::RobotLoader::register_object("JVRC1Fixed", fn_t([]() { return new JVRC1RobotModule(true); })); 
      return true;                                                                     
    }();                                                                               
}
#endif